sourcegraph/cody
# Edit/Chat: Change shortcut and add dedicated setting

Description:
- Adds a dedicated setting for "Command Hints". Disabled by default for now
- Changes the shortcut to "Opt+K" and "Opt+L" for Edit / Chat

<img width="424" alt="image" src="https://github.com/sourcegraph/cody/assets/9516420/25ccc07b-d504-4bfa-ad3d-e79982b77f68">

<img width="661" alt="image" src="https://github.com/sourcegraph/cody/assets/9516420/70b2534d-6ad8-4984-96f2-98ad5f9f0b53">

## Test plan

1. Enable "Command Hints"
2. Check hint text shows up on selections
3. Check shortcuts work for both Edit and Chat
4. Disable "Command Hints"
5. Check hint text doesn't show
6. Check shortcuts still work
